### ISSUE TITLE
[BITV/A11Y] Accessibility improvements

### ISSUE THREAD

**Issue by st3iny** [nextcloud/calendar#3540]
The accessibility of the calendar app should be improved. Here is a list of things that could be done.

* [x] #3955 (important)
* [ ] https://github.com/nextcloud/calendar/issues/4123 (important)
* [ ] https://github.com/nextcloud/calendar/issues/3522 (important)
* [x] #3945
* [ ] #3965
* [x] #3934
* [x] #3966
* [ ] https://github.com/nextcloud/calendar/issues/4128
* [ ] https://github.com/nextcloud/calendar/issues/4127
* [ ] https://github.com/nextcloud/calendar/issues/4126
* [ ] "New calendar" flow:
  * [ ] #4029
  * [x] #3932
  * [ ] https://github.com/nextcloud/calendar/issues/4125
* [ ] https://github.com/nextcloud/calendar/issues/4124

**Comment by tcitworld** [comment]
FullCalendar 5.10 seems to have a11y improvements https://github.com/fullcalendar/fullcalendar/blob/master/CHANGELOG.md#v5100-2021-10-13

**Comment by jancborchardt** [comment]
Sorted the most important 3 (bold) issues on top – as per judging from the notes document.

**Comment by ChristophWurst** [comment]
I've broken down the remaining items into separate tickets because the bugs are mostly independent.

The new tag (copied from server) allows us to find all accessibility tickets: https://github.com/nextcloud/calendar/labels/feature%3A%20accessibility.

**Comment by AndyScherzinger** [comment]
please don't close it before all subtasks have been finished
